src/Middleware/CasProxyTicketSessionMiddleware.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\Middleware;
+
+use Http\Promise\Promise;
+use Phpro\SoapClient\Middleware\Middleware;
+use Phpro\SoapClient\Middleware\MiddlewareInterface;
+use Psr\Http\Message\RequestInterface;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+
+/**
+ * Class CasProxyTicketSessionMiddleware.
+ */
+class CasProxyTicketSessionMiddleware extends Middleware implements MiddlewareInterface
+{
+    public const PGT_ATTRIBUTE = 'cas_pgt';
```

Comment:
Properties should be protected, we should provide setters/getters if necessary.

Reply:
Done. For now we won't need setters/getters, I skip adding them. Tell me if you think I should already add them, then we would have a property instead of a constant.